### ISSUE TITLE
Gun DLC

### ISSUE THREAD

**Issue by legos1819** [0x000015/SpaceEngineers-DLC-Bypass#21]
Will it support the new Warfare dlc?

**Comment by Stin045** [comment]
I agree it would be great to use it because i noticed it isnt unlocked yet

**Comment by Stin045** [comment]
could be also skin/emote unlock possible ?

**Comment by Lamer87** [comment]
https://github.com/Lamer87/Space-Engineers-DLC-unlocker
This is my work, it's the same thing but UPDATED ;)

PS: unlocking skins is impossible.
